script/deploy/single/06_DeployRolesAuthority.s.sol

```diff
@@ -27,6 +22,32 @@ contract DeployRolesAuthority is BaseScript {
         return deploy(getConfig());
     }
 
+    // Setup initial roles configurations
+    // --- Roles ---
```

Comment:
Comments here are off. May be able to just remove them completely if we now have them in constants:
uint8 constant STRATEGIST_ROLE = 1;
uint8 constant MANAGER_ROLE = 2;
uint8 constant TELLER_ROLE = 3;
uint8 constant UPDATE_EXCHANGE_RATE_ROLE = 4;
uint8 constant SOLVER_ROLE = 5;
uint8 constant PAUSER_ROLE = 6;

Reply:
Updated with all numbered roles